oci/private/util.bzl

```diff
@@ -42,15 +42,23 @@ def _parse_image(image):
     return (scheme, registry, repository, digest, tag)
 
 def _sha256(rctx, path):
-    """Returns hashsum of file at path
+    """Returns SHA256 hashsum of file at path
 
     Args:
         rctx: repository context
         path: path to the file
     Returns:
         hashsum of file
     """
-    result = rctx.execute(["shasum", "-a", "256", path])
+    # Attempt to use the first viable method to calculate the SHA256 sum. sha256sum is part of
+    # coreutils on Linux, but is not available on MacOS. shasum is a perl script that is available
+    # on MacOS, but is not necessarily always available on Linux. OpenSSL is used as a final
+    # fallback if neither are available
+    result = rctx.execute(["sha256sum", path])
```

Comment:
for performance reasons; can have the fallback strategy in this order; shasum256 is widely available on macos and linux distros, so it should be the first.

1- shasum
2- sha256sum
3- openssl